roles/confluent.common/tasks/redhat.yml

```diff
@@ -40,13 +40,10 @@
   command: yum clean all
   args:
     warn: false
-  when: confluent_dist_repo_result.changed|default(False) or confluent_repo_result.changed|default(False) or custom_repo_result.changed|default(False) or repository_configuration == 'custom'
-
-- name: yum-makecache
```

Comment:
So it seems this PR: https://github.com/confluentinc/cp-ansible/pull/786/files resolved this issue: https://github.com/confluentinc/cp-ansible/issues/785

But is introducing an issue for you @erikgb ... 

I'm not sure what the most diplomatic way of resolving this is then. 

Think we could make use of something like:
```
yum --disablerepo="*" --enablerepo="repoA" update
```

Reply:
My hope is that this PR should not "un-fix" #785.